themes/default/public/styles/raneto.css

```diff
@@ -332,15 +393,15 @@
 
 a:hover
 {
-    color: #026288;
+    color: #dae611;
```

Comment:
@willmero instead of hardcoding these values, lets follow best practice and set them in a variable?